docs/content/configuration/index.md

```diff
@@ -417,11 +417,13 @@ JavaScript-based functionality is disabled by default. Please refer to the Druid
 
 ### Double Column storage
 
-Druid's storage layer uses a 32-bit float representation to store columns created by the 
-doubleSum, doubleMin, and doubleMax aggregators at indexing time. To instead use 64-bit floats
-for these columns, please set the system-wide property `druid.indexing.doubleStorage=double`.
-This will become the default behavior in a future version of Druid.
+Prior to version `0.13.0` Druid's storage layer uses a 32-bit float representation to store columns created by the 
```

Comment:
Grammar / formatting:

- "used a 32-bit", not uses
- I'd say don't backtick-format 0.13.0
- borderline on whether doubleSum/doubleMin/doubleMax should be backtick-formatted; I probably wouldn't
- "keep the old format" not "keep old format"
- I think it'd be clearer to replace the last sentence with "Support for 64-bit floating point columns was released in Druid 0.11.0, so if you use this feature then older versions of Druid will not be able to read your data segments."